index.js

```diff
@@ -1,66 +1,187 @@
-module.exports = {
-    "extends": ["airbnb-base"],
-    "plugins": [
-        "import"
-    ],
-    "env": {
-        "node": true,
-        "browser": true
+const globals = require('globals');
+const js = require('@eslint/js');
+const typescriptEslint = require('typescript-eslint');
+
+const {
+    FlatCompat,
+} = require('@eslint/eslintrc');
+
+const compat = new FlatCompat({
+    baseDirectory: __dirname,
+    recommendedConfig: js.configs.recommended,
+    allConfig: js.configs.all
+});
+
+module.exports = [...compat.extends('airbnb-base'),
+    {
+        linterOptions: {
+            reportUnusedDisableDirectives: true,
+        },
+
+        languageOptions: {
+            globals: {
+                ...globals.node,
+                ...globals.browser,
+            },
+
+            ecmaVersion: 2022,
+        },
+
+        rules: {
+            indent: ['error', 4, {
```

Comment:
Yeah, formatting will be the next step, but it would make scope of the project too large, so I'll keep it here until we move forward with `prettier` in the future

Reply:
yes, my point is we should not use this rule, not that we should add prettier right ahead. its broken and console team will have issues with that in the nest repo most likely.